scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContextVariableExtractor.java

```diff
@@ -147,6 +147,30 @@
         return VariableSubstitutor.resolveVariables(variables, dictionary);
     }
 
+    /**
+     * Method to retrieve job results variables in scope.
+     *
+     * @param taskContext task context container.
+     *
+     * @return Map containing all job results variables extracted.
+     *
+     */
+    public Map<String, Serializable> getJobMapVariables(TaskContext taskContext) {
+        Map<String, Serializable> JobMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)